Overview.src.html

```diff
@@ -830,6 +842,126 @@ <h2 class="no-num no-toc">
       </section>
       <section>
         <h3>
+          Interface <code>Availability</code>
+        </h3>
+        <pre class="idl">
+interface <dfn>Availability</dfn> : EventTarget {
+  readonly attribute boolean value;
+  attribute <span data-anolis-spec=
+"w3c-html">EventHandler</span> <span>onchange</span>;  
+};
+</pre>
+        <p>
+          The <code>change</code> event is fired during the execution of the
+          <span>monitoring presentation display availability</span> algorithm
+          when the <dfn>presentation display availability</dfn> changes. It is
+          fired at the <span><code>Availability</code></span> object, using the
+          <code>ChangeEvent</code> interface. The <code>Availability</code>
+          object's <code>value</code> attribute is updated to the boolean value
+          that the algorithm determined.
+        </p>
+        <p>
+          In order to satisfy the <a href=
+          "https://github.com/w3c/presentation-api/blob/gh-pages/uc-req.md#req08-power-saving-friendly">
+          power saving non-functional requirement</a>, the user agent must keep
+          track of the number of <code>Availability</code> objects requested
+          through the <code>getAvailability()</code> method. The UA may also
+          keep track of whether the page holding the <code>Availability</code>
+          object is in the foreground. Using this information, implementation
+          specific discovery of <span title="presentation-display">presentation
+          displays</span> can be resumed or suspended, in order to save power.
+        </p>
+        <p>
+          The user agent must keep a <dfn>list of available presentation
+          displays</dfn>. According to the number of alive
```

Comment:
We should define what it means for an Availability object to be "alive" - I assume "not eligible for garbage collection."   I wonder if there is a standard definition for this?